tgui/packages/tgui/interfaces/GasAnalyzer.tsx

```diff
@@ -29,11 +54,85 @@ export const GasAnalyzerContent = (props, context) => {
   );
 };
 
+export const GasAnalyzerHistory = (props, context) => {
+  const { act, data } = useBackend<GasAnalyzerHistoryData>(context);
+  const { historyGasmixes, historyViewMode, historyIndex } = data;
+  return (
+    <Section
+      scrollable
+      fill
+      title="Scan History"
+      buttons={
+        <Button
+          icon={'trash'}
+          tooltip="Clear History"
+          onClick={() => act('clearhistory')}
+          textAlign="center"
+          disabled={historyGasmixes.length === 0}
+        />
+      }>
+      <LabeledList.Item label="Mode">
+        <Flex inline width="50%">
+          <Flex.Item>
+            <Button
+              content={'kPa'}
+              onClick={() => act('modekpa')}
+              textAlign="center"
+              selected={historyViewMode === 'kpa'}
+            />
+          </Flex.Item>
+          <Flex.Item>
+            <Button
+              content={'mol'}
```

Comment:
do not use content, use put the text as a child

Reply:
What do you mean exactly? I'm kind of new to TGUI